Rectangle.java

```diff
@@ -0,0 +1,38 @@
+public class Rectangle {
+    private int hight;
+    private int width;
+
+    public Rectangle(){
+

```

Comment:
What is the purpose of empty constructor? What values are going to be used for height' and 'width' variables?